### ISSUE TITLE
Submit button cannot be chosen when there are more than 2 buttons

### ISSUE THREAD

**Issue by giorgiosironi** [hickford/MechanicalSoup#61]
From https://github.com/hickford/MechanicalSoup/blob/master/mechanicalsoup/form.py#L131:
```
        for inp in self.form.select("input"):
            if inp.get('type') != 'submit':
                continue
            if inp == el:
                continue

            del inp['name']
            return True
```
When there are 3 or more submits and one is chosen, only one is deleted and then the method early returns.

**Comment by moy** [comment]
There's definitely room for improvement here, but there are already workarounds. See #65 and in particular [this comment](https://github.com/hickford/MechanicalSoup/issues/65#issuecomment-289011578).

**Comment by dbrobins** [comment]
Can this be closed with #70?

**Comment by moy** [comment]
Indeed. Next time, you can add "fixes #number" to the commit message to get the bug referenced and closed automatically. 

Thanks,
